File: a2aNetwork/sdk/typescript/src/constants/networks.ts
export const NETWORKS: Record<string, {
  chainId: number;
  name: string;
  rpcUrls: string[];
  currency: {
    name: string;
    symbol: string;
    decimals: number;
  };
}> = {
  localhost: {
    chainId: 31337,
    name: 'Localhost',
    rpcUrls: ['http://localhost:8545'],
    currency: {
      name: 'Ether',
      symbol: 'ETH',
      decimals: 18
    }
  },
  mainnet: {
    chainId: 1,
    name: 'Ethereum Mainnet',
    rpcUrls: [process.env.MAINNET_RPC_URL || 'https://cloudflare-eth.com'],
    currency: {
      name: 'Ether',
      symbol: 'ETH',
      decimals: 18
    }
  },
  sepolia: {
    chainId: 11155111,
    name: 'Sepolia Testnet',
    rpcUrls: [process.env.SEPOLIA_RPC_URL || 'https://rpc.sepolia.org'],
    currency: {
      name: 'Sepolia Ether',
      symbol: 'ETH',
      decimals: 18
    }
  },
  polygon: {
    chainId: 137,
    name: 'Polygon Mainnet',
    rpcUrls: ['https://polygon-rpc.com'],
    currency: {
      name: 'MATIC',
      symbol: 'MATIC',
      decimals: 18
    }
  }
};

export type NetworkName = keyof typeof NETWORKS;

export const DEFAULT_NETWORK: NetworkName = 'localhost';

File: a2aNetwork/sdk/python/a2a/constants/networks.py
"""
A2A Network Constants - Networks

Network configuration constants.
"""

NETWORKS = {
    'mainnet': {
        'chain_id': 1,
        'rpc_urls': ['https://mainnet.infura.io/v3/'],
        'poa': False
    },
    'goerli': {
        'chain_id': 5,
        'rpc_urls': ['https://goerli.infura.io/v3/'],
        'poa': True
    },
    'sepolia': {
        'chain_id': 11155111,
        'rpc_urls': ['https://sepolia.infura.io/v3/'],
        'poa': False
    },
    'localhost': {
        'chain_id': 31337,
        'rpc_urls': ['http://localhost:8545'],
        'poa': False
    }
}

DEFAULT_NETWORK = 'localhost'

__all__ = ['NETWORKS', 'DEFAULT_NETWORK']